lib/galaxy/tools/errors.py

```diff
@@ -125,28 +128,37 @@
 ${job_traceback}
 </pre>
 
+<h4>Tool Transcript</h4>
+<pre style="white-space: pre-wrap;background: #eeeeee;border:1px solid black;padding:1em;">
+${tool_transcript}
+</pre>
+
```

Comment:
The report type thing is good I think - I think we should push it down a level though. These templates are going to have very different things filled in based on the report type right? I think it would simplify what we present to user support and to the users to have a different template for each report type. That way we know we're only displaying useful information for the relevant report type. Not displaying job details if no job was created for instance.

Reply:
Your insight is helpful as this particular legacy code block was a sticking point.